QSB/EchoesOfTheEye/AirlockSync/WorldObjects/QSBGhostAirlock.cs

```diff
@@ -2,7 +2,9 @@
 
 namespace QSB.EchoesOfTheEye.AirlockSync.WorldObjects
 {
+	// unused?
```

Comment:
will be used in EOTE stuff, leftover from when i was testing out those initial syncs